ccf_transaction_processor/transaction_processor/pdo_tp.cpp

```diff
@@ -51,6 +54,29 @@ namespace ccfapp
             return ccf::make_success(true);
         };
 
+        //======================================================================================================
+        // register enclave expected measurements (member method)
+        auto register_enclave_expected_measurements = [this](auto& ctx, const nlohmann::json& params) {
```

Comment:
Some functions throughout the TP have explicitly typed params, while others like this one use `auto` types. I'm personally not a fan of `auto` types in general, but we should be consistent, especially in function declarations. Also makes the code considerably more readable, imo :)